nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java

```diff
@@ -1540,6 +1540,12 @@ public boolean isZooKeeperTlsConfigurationPresent() {
             && getProperty(NiFiProperties.ZOOKEEPER_SECURITY_TRUSTSTORE_PASSWD) != null;
     }
 
+    public boolean isTlsConfigurationPresent() {
```

Comment:
We make a stronger assumption in `isZooKeeperTlsConfigurationPresent()` that the truststorePasswd is not null. That's not included here which is I think the correct thing. It might be worth changing `isZooKeeperTlsConfigurationPresent()` to align with `isTlsConfigurationPresent()` or vice versa.